Makefile.js

```diff
@@ -159,7 +159,8 @@ function execSilent(cmd) {
  * @private
  */
 function generateBlogPost(releaseInfo) {
-    const output = ejs.render(cat("./templates/blogpost.md.ejs"), releaseInfo),
+    const ruleList = ls("lib/rules").map(ruleFileName => ruleFileName.replace(/\.js$/, "")).sort((ruleA, ruleB) => ruleB.length - ruleA.length);
```

Comment:
I think `path.basename` requires you to provide an extension, otherwise it doesn't separate the extension properly.

If we want to account for different file extensions, we could do something like `.replace(/\..*?$/, '')`